dlpy/model.py

```diff
@@ -699,6 +699,10 @@ def train(self, table, attributes=None, inputs=None, nominals=None, texts=None,
                               force_equal_padding=force_equal_padding, data_specs=data_specs, n_threads=n_threads,
                               target_order=target_order)
 
+        # the recordSeed option must not be specified in order to disable it, contrary to the Viya documentation
+        if record_seed == 0 or record_seed == None:
```

Comment:
I missed one more thing. would you also please change 'record_seed == None' to 'record_seed is None'
Thanks

Reply:
Pushed the requested change.